### ISSUE TITLE
Unable to use in the latest version

### ISSUE THREAD

**Issue by frilxy** [MoshirMoshir/Moshir-OpenTaiko-Skin#4]
Unable to use in the latest version, selecting this skin prevents entering the game.

**Comment by MoshirMoshir** [comment]
Yeah, there's some weird issues now, I'd have to check and update the skin.

I'll be working on it either this or next weekend.
